src/NovelLoggingService.h

```diff
@@ -25,14 +25,36 @@ class NovelLoggingService {
   NovelLoggingService(const std::string& core, LogLevel level);
   void log(const std::string& message, LogLevel level);
   void logInfoLine(const std::string& message);
+  void logInfo();
   void logErrorLine(const std::string& message);
-  template <typename T, typename ... TRest> void logInfo(T current, TRest ... next);
-  template <typename E, typename ... ERest> void logError(E current, ERest ... next);
-  void logWarning(const std::string& message);
-  void logDebug(const std::string& message);
+  void logError();
```

Comment:
nit: These are only for terminating the variadic template. They can be private and inline which should help codegen and prevent people from calling them.